helm/test-values.yaml

```diff
@@ -0,0 +1,249 @@
+  #      favicon: https://raw.githubusercontent.com/iteratec/multi-juicer/main/wrongsecrets-balancer/ui/public/favicon.ico
+  #      showVersionNumber: false
+  #      showGitHubLinks: false
+  #    challenges:
+  #      showHints: true
+  #    hackingInstructor:
+  #      isEnabled: true
+  #    ctf:
+  #      showFlagsInNotifications: false
+  # -- Specify a custom NODE_ENV for JuiceShop. If value is changed to something other than 'wrongsecrets-ctf-party' it's not possible to set a custom config via `juiceShop.config`.
+  nodeEnv: "wrongsecrets-ctf-party"
+  # -- Optional resources definitions to set for each JuiceShop instance
+  resources:
+    requests:
+      cpu: 256Mi
+      memory: 300Mi
+  #  limits:
+  #    cpu: 100m
+  #    memory: 200Mi
+  # -- Optional securityContext definitions to set for each JuiceShop instance
+  securityContext:
+    allowPrivilegeEscalation: false
+    readOnlyRootFilesystem: true
+    runAsNonRoot: true
+    capabilities:
+      drop:
+        - ALL
+    seccompProfile:
+      type: RuntimeDefault
+  # -- Optional environment variables to set for each JuiceShop instance (see: https://kubernetes.io/docs/tasks/inject-data-application/define-environment-variable-container/)
+  env:
+    - name: K8S_ENV
+      value: k8s
+    - name: SPECIAL_K8S_SECRET
+      valueFrom:
+        configMapKeyRef:
+          name: secrets-file
+          key: funny.entry
+    - name: SPECIAL_SPECIAL_K8S_SECRET
+      valueFrom:
+        secretKeyRef:
+          name: funnystuff
+          key: funnier
+  # env:
+  #   - name: FOO
+  #     valueFrom:
+  #       secretKeyRef:
+  #         key: FOO
+  #         name: secret-resource
+  # -- Optional mount environment variables from configMaps or secrets (see: https://kubernetes.io/docs/tasks/inject-data-application/distribute-credentials-secure/#configure-all-key-value-pairs-in-a-secret-as-container-environment-variables)
+  envFrom: []
+  # -- Optional Volumes to set for each JuiceShop instance (see: https://kubernetes.io/docs/concepts/storage/volumes/)
+  volumes: []
+  # create config map with a custom logo via: kubectl create configmap custom-logo --from-file custom.png=your-logo.png
+  # then switch out the logo parameter in the wrongsecrets config section above to the mounted filename.
+  # volumes:
+  # - name: logo
+  #   configMap:
+  #     name: custom-logo
+  # -- Optional VolumeMounts to set for each JuiceShop instance (see: https://kubernetes.io/docs/concepts/storage/volumes/)
+  volumeMounts: []
+  # volumeMounts:
+  # - name: logo
+  #   mountPath: /wrongsecrets/frontend/dist/frontend/assets/public/images/custom.png
+  #   subPath: custom.png
+  #   readOnly: true
+
+  # -- Optional Configure kubernetes scheduling affinity for the created JuiceShops (see: https://kubernetes.io/docs/concepts/scheduling-eviction/assign-pod-node/#affinity-and-anti-affinity)
```

Comment:
This can be removed and/or host values we have in code for the storage configuration

Reply:
Done

Reply:
I noticed it will result in this change in the configmap for the wrongsecrets balancer
https://github.com/OWASP/wrongsecrets-ctf-party/pull/249/files#diff-0e79142774e4871080c08cade4299253e8b84c26fab30421555d6a454406abf7R70
If the wrongsecrets is checking first if the `volumeMounts` section is `null` that should be ok.